web/private/scripts/drush_post_deploy_config_import.php

```diff
@@ -8,5 +8,5 @@
 
 // Import all config changes.
 echo "Start importing configuration from yml files...\n";
-system('drush config-import -y 2>&1');
+system('drush config-import -y -vvv 2>&1');
```

Comment:
Do you think it would be helpful to output a timestamp before and after the config import? (So that we'll know the exact time it starts and finishes, might be helpful if we need to correlate with external events.) 

Something like `system('date +"%T"');` should do the trick.

Reply:
That's a great idea. I've added and pushed. Could you give it another review when the build finishes?